utils/email_service.py

```diff
@@ -1,17 +1,55 @@
 from celery import shared_task
 from decouple import config
-from django.core.mail import EmailMessage
+from django.core.mail import EmailMessage, send_mail
 from decouple import config
+import boto3
 
 
 @shared_task
-def send_email(recipient,subject,body):
-    if config('MOCK_EMAIL', cast=bool, default=True):
+def send_email(recipient, subject, body):
+    if config('MOCK_EMAIL', cast=bool):

```

Comment:
Here you can use f string, It would be more easy and better to look at than multiple print statements.